Features/Context/PlatformUI.php

```diff
@@ -106,6 +102,22 @@ public function __construct($uri = self::PLATFORM_URI, $user = null, $password =
     /**
      * @BeforeScenario
```

Comment:
This would need some doc.
something in the lines of:
```
 *
 * Parses Context properties typehints and automatically assigns them the context instances, if any.
 */
```